frontend/src/components/auth/Login.tsx

```diff
@@ -51,7 +45,7 @@ const Login = (): React.ReactElement => {
       tokenId,
     );
 
-    if (isAuthenticatedUserType(res)) {
+    if (res) {
```

Comment:
also think we can just check `res` exists here instead of having the whole function extracted right?